lib/strings.c

```diff
@@ -126,3 +127,25 @@ ktxSupercompressionSchemeString(ktxSupercmpScheme scheme)
             return "Vendor scheme";
     }
 }
+
+/**
+* @~English
+* @brief Return a string corresponding to a bu_image_flags bit.
+*
+* @param bit_index the bu_image_flag bit to test.
+* @param bit_value the bu_image_flag bit value.
+*
+* @return pointer to the message string or NULL otherwise.
+*
+* @internal Use UTF-8 for translated message strings.
+*/
+const char* ktxBUImageFlagsBitString(ktx_uint32_t bit_index, bool bit_value)
+{
+    if (!bit_value)
+        return NULL;
+
+    switch (1u << bit_index) {
+        case eBUImageIsPframe: return "eBUImageIsPframe";
```

Comment:
The spec calls this flag `isPFrame`. We should probably rename it everywhere to something different using uppercase for consistency with other enums, like `ETC1S_P_FRAME`. Thoughts?

Reply:
I can rename it, `ETC1S_P_FRAME` seams like a more appropriate name for me too.